src/nvim/options.lua

```diff
@@ -496,12 +496,12 @@ return {
           use '//', instead of '\\'.
         - Environment variables are expanded |:set_env|.
         - Careful with '\' characters, type one before a space, type two to
-          get one in the option (see |option-backslash|), for example: >
-            :set bdir=c:\\tmp,\ dir\\,with\\,commas,\\\ dir\ with\ spaces
+          get one in the option (see |option-backslash|), for example: >vim
+             set bdir=c:\\tmp,\ dir\\,with\\,commas,\\\ dir\ with\ spaces
```

Comment:
```suggestion
            set bdir=c:\\tmp,\ dir\\,with\\,commas,\\\ dir\ with\ spaces
```

Reply:
These indentations (and space vs tab in general) are not consistent.

Reply:
But it is usually not an odd number :)

Reply:
sw=3 best sw :)